app/actions/views/login.js

```diff
@@ -31,6 +34,12 @@ export function handleSuccessfulLogin() {
         const token = Client4.getToken();
         const url = Client4.getUrl();
 
+        const enableTimezone = config.ExperimentalTimezone === 'true';
```

Comment:
Can we put this in a helper function? We're repeating this logic quite a few times